asf_heat_pump_suitability/pipeline/run_scripts/run_add_features.py

```diff
@@ -139,28 +144,23 @@ def parse_arguments() -> argparse.Namespace:
     logging.info("Adding property density to EPC")
     epc_df = property_density.extend_df_with_property_density(epc_df)
 
+    # Add feature: property density Scotland
+    dz_density_df = property_density.generate_df_property_density_s()
+    epc_df = epc_df.join(dz_density_df, how="left", left_on="lsoa", right_on="DataZone")
```

Comment:
I have two columns - `households_per_km2` and `households_per_km2_right` when I ran this script, I think this join is where it happened. I assume they'll need to be merged into one?

Reply:
Great spot thank you!